tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -9,7 +9,6 @@
 class Search(Base):
 
     name = 'Browser'
-    manifest_url = "app://search.gaiamobile.org/manifest.webapp"
```

Comment:
Yes, you have right it's search I checked now : 
DOMApplication { manifest: Object, updateManifest: null, manifestURL: "app://search.gaiamobile.org/manifes…", origin: "app://search.gaiamobile.org", installOrigin: "app://search.gaiamobile.org", installTime: 1440974431138, removable: false, enabled: true, receipts: Array[0], progress: 0 } 
Then we took the class name in this syntax: return '{}{}.{}/manifest.webapp'.format(self.DEFAULT_PROTOCOL, format(self.__class__.__name__).lower(), self.DEFAULT_APP_HOSTNAME)

so I don't think it's an issue